web/packages/teleport/src/services/integrations/types.ts

```diff
@@ -121,8 +152,9 @@ export type IntegrationSpecAwsOidc = {
   audience?: IntegrationAudience;
 };
 
+// IntegrationSpecAwsRa contain the specific fields for the `aws-ra` subkind integration. [go struct ui.IntegrationAWSRASpec]
 export type IntegrationSpecAwsRa = {
-  trustAnchorArn: string;
+  trustAnchorARN: string; // ARN per API json tag
```

Comment:
i honestly dont know which version of capitalization to use. i think its `trustAnchorArn`, but whatever it is, we should keep it consistent